README.md

```diff
@@ -51,14 +51,27 @@ A **Model Serialization Attack** can be used to execute:
 
 These attacks are incredibly simple to execute and you can view working examples in our 📓[notebooks](https://github.com/protectai/modelscan/tree/main/notebooks) folder.
 
+## Scanning Models At Scale
```

Comment:
Claude suggested re-write below. Won't recommend using it verbatim but I like it!

# Elevate Your AI Security with Guardian

While ModelScan offers robust open-source scanning, organizations seeking comprehensive AI security should consider [Guardian](https://protectai.com/guardian), our enterprise-grade solution.

![Guardian Overview](/imgs/guardian_overview.png)

## Guardian: Advanced Protection for Enterprise AI

Guardian takes your AI security to the next level:

1. **Cutting-Edge Scanning**: Access our latest scanners, broader model support, and automatic model format detection.

2. **Proactive Security**: Define and enforce security requirements for Hugging Face models before they enter your environment—no code changes required.

3. **Enterprise-Wide Coverage**: Implement a cohesive security posture across your organization, seamlessly integrating with your CI/CD pipelines.

4. **Comprehensive Audit Trail**: Gain full visibility into all scans and results, empowering you to identify and mitigate threats effectively.

Ready to fortify your AI infrastructure? [Explore Guardian today](https://protectai.com/guardian) and safeguard your organization's AI assets.